Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "iota-crypto"
-version = "0.6.0"
+version = "0.7.0"
```

Comment:
Please don't do this. We automatically bump 
Package versions with covector.

Reply:
Sorry about that. Done